### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by weffke**
CertiK to Shentu modification

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,44 +148,44 @@ seeds = "f4678480e4b7f1daee8ff47e6265954f8b57291d@54.234.180.96:26656,f6764b2a92
 ...
 ```
 
-Then copy the testnet genesis JSON file to the node configuration directory. The latest genesis file can be found [here](https://github.com/certikfoundation/chain/blob/master/genesis.json). Release-specific genesis file can be found in the [release download](https://github.com/certikfoundation/chain/releases/).
+Then copy the testnet genesis JSON file to the node configuration directory. The latest genesis file can be found [here](https://github.com/shentufoundation/chain/blob/master/genesis.json). Release-specific genesis file can be found in the [release download](https://github.com/shentufoundation/chain/releases/).
```
Comment (haozhan9) [inline README.md:151]: the links here should be
https://github.com/shentufoundation/shentu/blob/master/genesis.json
https://github.com/shentufoundation/shentu/releases/

File: README.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ It is the only way to recover your account if you ever forget your password.
 ...
 ```
 
-Then, go to http://explorer.certik.foundation/faucet, submit your test account's address. The account will receive some amount of CTK for testing purpose. Please do not abuse the faucet as there are only limited number of tokens available for testnet.
+Then, go to http://explorer.shentu.foundation/faucet, submit your test account's address. The account will receive some amount of CTK for testing purpose. Please do not abuse the faucet as there are only limited number of tokens available for testnet.
```
Comment (0311xuyang) [inline README.md:120]: url domain should be `explorer.shentu.technology`

File: README.md [hunk 3]
```diff
@@ -69,43 +69,43 @@ The [Shentu Chain Explorer](https://explorer.shentu.foundation/) is public acces
...
 
-For non-public chain questions and discussions, please email to chain@certik.org
+For non-public chain questions and discussions, please email to info@Shentu.org
 
 Chain transactions as well as chain account creation can also be performed via the CLI tool or CLI-based RESTful servers.
 
 ## Latest Release Download
 
-The latest chain node binary and CLI tool binary, as well as genesis and sample smart contract files, can be downloaded from https://github.com/certikfoundation/chain/releases/.
+The latest chain node binary and CLI tool binary, as well as genesis and sample smart contract files, can be downloaded from https://github.com/shentufoundation/chain/releases/.
```
Comment (0311xuyang) [inline README.md:82]: link should be `https://github.com/shentufoundation/shentu/releases/`

File: README.md [hunk 4]
```diff
@@ -69,43 +69,43 @@ The [Shentu Chain Explorer](https://explorer.shentu.foundation/) is public acces
 
 The [DeepWallet](https://wallet.shentu.foundation/) is public accessible and can be used to manage chain accounts, including CTK storage, staking, transfer, and voting, as well as smart contracts deploy and invocation.
```
Comment (0311xuyang) [inline README.md:70]: The Deepwallet link is `https://wallet.shentu.technology/`

File: README.md [hunk 5]
```diff
@@ -69,43 +69,43 @@ The [Shentu Chain Explorer](https://explorer.shentu.foundation/) is public acces
...
 
-The [Certik Chain User Forum](https://forum.certik.foundation/categories) is a public forum for chain announcements and discussions.
+The [Shentu Chain User Forum](https://forum.shentu.foundation/categories) is a public forum for chain announcements and discussions.
```
Comment (0311xuyang) [inline README.md:72]: The forum link is out-dated.

File: README.md [hunk 6]
```diff
@@ -69,43 +69,43 @@ The [Shentu Chain Explorer](https://explorer.shentu.foundation/) is public acces
...
-certik config node tcp://<full node>:<port>
-certik config trust-node true
+rm -rf ~/.shentud
+shentud config chain-id <chain-id>
+shentud config node tcp://<full node>:<port>
+shentud config trust-node true
 ```
 
-`certik` runs on Linux, Windows, and MacOS.
+`Shentu` runs on Linux, Windows, and MacOS.
```
Comment (0311xuyang) [inline README.md:97]: typo, `Shentu` -> `shentud`

File: README.md [hunk 7]
```diff
@@ -148,44 +148,44 @@ seeds = "f4678480e4b7f1daee8ff47e6265954f8b57291d@54.234.180.96:26656,f6764b2a92
...
 
 First, run the following on on your node instance.
 
 ```
-certik tendermint show-validator
+shentud tendermint show-validator
 ```
 
-Then send the pubkey to chain@certik.org with title "request to certify validator node" and optionally your reasoning on why this node should be approved to become a validator node (typically in terms of capacity, connectivity, maintenance, and security).
+Then send the pubkey to info@shentu.org with title "request to certify validator node" and optionally your reasoning on why this node should be approved to become a validator node (typically in terms of capacity, connectivity, maintenance, and security).
```
Comment (0311xuyang) [inline README.md:179]: Is the email address confirmed ?

File: README.md [hunk 8]
```diff
@@ -197,59 +197,59 @@ certik tx staking create-validator \
   --from=<validator chain account name>
 ```
 
-If the transaction is successful, your validator should appear in either the `Active` or `Inactive` tabs on the [chain explorer's validators page](https://explorer.certik.foundation/validators).
+If the transaction is successful, your validator should appear in either the `Active` or `Inactive` tabs on the [chain explorer's validators page](https://explorer.shentu.foundation/validators).
```
Comment (0311xuyang) [inline README.md:200]: `explorer.shentu.foundation` -> `explorer.shentu.technology`

File: README.md [hunk 9]
```diff
@@ -197,59 +197,59 @@ certik tx staking create-validator \
...
 
 ```
-certik query account <address>
+shentud query account <address>
 ```
 
 ### Transactions
 
-`certik` also supports many transaction subcommands. Below are some common examples. More details can be found in the command help printout.
+`Shentu` also supports many transaction subcommands. Below are some common examples. More details can be found in the command help printout.
```
Comment (0311xuyang) [inline README.md:234]: shentud

File: README.md [hunk 10]
```diff
@@ -324,10 +324,10 @@ data: 000000000000000000000000000000000000000000000000000000000000007B
 To start a local RESTful server connected to the full node connected above
 
 ```
-certik rest-server
+shentud rest-server
 ```
 
-you can look through the swagger documents through `localhost:1317/swagger/` while the `certik rest-server` is running.
+you can look through the swagger documents through `localhost:1317/swagger/` while the `Shentu rest-server` is running.
```
Comment (0311xuyang) [inline README.md:330]: shentud

**Pull request merged**